templates/esm/typescript_gapic/esm/src/$version/$service_client.ts.njk

```diff
@@ -607,6 +608,9 @@ export class {{ service.name }}Client {
 {%- endif %}
       ]>|void {
     request = request || {};
+    {% for field in method.autoPopulatedFields %}if (!request.{{ field.toCamelCase() }}) {
+      request.{{ field.toCamelCase() }} = crypto.randomUUID();
```

Comment:
It does not seem like this is covered in any baseline tests - we should have a baseline that would contain this line.